src/backend/TrafficCourts/Citizen.Service/Program.cs

```diff
@@ -1,14 +1,20 @@
+using Serilog;
+using System.ComponentModel;
 using System.Reflection;
-using System.Text.Json.Serialization;
 using TrafficCourts.Citizen.Service;
 using TrafficCourts.Common.Configuration;
+using TrafficCourts.Common.Configuration.Validation;
+using TrafficCourts.Common.Utils;
 
 WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+Serilog.ILogger logger = GetLogger(builder);
 
-builder.ConfigureApplication(); // this can throw ConfigurationErrorsException
+builder.ConfigureApplication(logger); // this can throw ConfigurationErrorsException
 
 // Add services to the container.
-builder.Services.AddControllers();
+builder.Services
+    .AddControllers(options => options.UseDateOnlyTimeOnlyStringConverters())
+    .AddJsonOptions(options => options.UseDateOnlyTimeOnlyStringConverters());
```

Comment:
Should you also add JsonStringEnumConverter? I had to add that on the staff-api so enum are serialized correctly (as strings instead of ints).
```suggestion
    .AddJsonOptions(options => options.UseDateOnlyTimeOnlyStringConverters())
    .AddJsonOptions(options => options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter())); 
```

Reply:
It is being added by the registered class `public class ConfigureJsonOptions : IConfigureOptions<JsonOptions>`. It not obvious. the framework will create all registered ` IConfigureOptions<JsonOptions>` types an calls their configured method. It would make sense to consolidate the configuration into a reusable config so each service uses same settings.